blazor/diagram/connectors/segments/orthogonal.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: Orthogonal Segment in Blazor Diagram Component | Syncfusion
+description: Checkout and learn here all about Orthogonal Segments in Syncfusion Blazor Diagram component and much more details.
```

Comment:
description: Check out and learn about Orthogonal Segments in Syncfusion Blazor Diagram component and much more details.